src/main/java/com/codecool/keepcash/Controller/TestController.java

```diff
@@ -0,0 +1,73 @@
+//package com.codecool.keepcash.Controller;
+//
+//import com.codecool.keepcash.Dto.Account.AccountTypeDto;
+//import com.codecool.keepcash.Dto.Currency.CurrencyDto;
+//import com.codecool.keepcash.Dto.Operation.OperationDto;
+//import com.codecool.keepcash.Service.Account.AccountTypeService;
+//import com.codecool.keepcash.Service.Currency.CurrencyService;
+//import com.codecool.keepcash.Service.Operation.OperationService;
+//import org.springframework.web.bind.annotation.*;
+//
+//import java.util.List;
+//
+//import static org.springframework.http.HttpStatus.CREATED;
+//import static org.springframework.http.HttpStatus.NO_CONTENT;
+//
+//@RestController
+//@RequestMapping("/api/v1")
+//public class TestController {
```

Comment:
1.We don't consider it production code but I understand that we should as it's on 'dev' branch.
2.It's called 'TestController' because it was our first Rest controller, so we put there all our methods for excercise purposes.